android-base/src/main/res/layout/activity_main.xml

```diff
@@ -59,6 +59,7 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@id/toolbar"
                 app:navGraph="@navigation/navigation"
+                tools:ignore="FragmentTagUsage"
```

Comment:
Thanks! 👍 
It looks better if comments are written with the ignoring attribute.

Reply:
Added!
Is this enough information?